python/tvm/script/parser/relax/parser.py

```diff
@@ -252,7 +253,7 @@ def visit_ann_assign(self: Parser, node: doc.AnnAssign) -> None:
     )
     var = self.var_table.get().get(lhs.id)
     assert isinstance(var, relax.Var)
-    R.ir.annotate_type_shape(var, ann_type, ann_shape)
+    R.ir.annotate_type_shape(var, ann_sinfo)
```

Comment:
This one is left to be renamed.
```suggestion
    R.ir.annotate_struct_info(var, ann_sinfo)
```